### ISSUE TITLE
input file error on 2.2.0

### ISSUE THREAD

**Issue by jamigo** [seppinho/haplogrep-cmd#34]
I'm currently testing v2.2.0, and I'm getting an input error:
```
$ haplogrep classify --format vcf --extend-report --in test.vcf --out test.txt

https://haplogrep.i-med.ac.at
Sebastian Schoenherr, Hansi Weissensteiner, Lukas Forer and Domininc Pacher
Institute of Genetic Epidemiology, Medical University of Innsbruck
Contact: sebastian.schoenherr@i-med.ac.at
Welcome to HaploGrep v2.1.25
Instiute of Genetic Epidemiology, Medical University of Innsbruck
Hansi Weissensteiner, Sebastian Schönherr, Lukas Forer, Dominic Pacher

Error. Please check if input file exists
```
previously, relative paths were supported. if this is no longer the case and absolute paths are now mandatory I'd suggest to include a side note to that "Please check if input file exists" message.

... plus check "Instiute" instead of "Institute", and "v2.1.25" instead of "v2.1.26" if needed.

**Comment by seppinho** [comment]
hi @jamigo 
Relative paths are still supported. Can you double check on your side? Fixed the typo. thanks.

**Comment by jamigo** [comment]
I've updated to latest 2.2.3, but unfortunately I still see the same error:

```
$ haplogrep classify --format vcf --extend-report --in test.vcf --out test.txt

https://haplogrep.i-med.ac.at
Sebastian Schoenherr, Hansi Weissensteiner, Lukas Forer and Domininc Pacher
Institute of Genetic Epidemiology, Medical University of Innsbruck
Contact: sebastian.schoenherr@i-med.ac.at
Welcome to HaploGrep v2.2.3
Instiute of Genetic Epidemiology, Medical University of Innsbruck
Hansi Weissensteiner, Sebastian Schönherr, Lukas Forer, Dominic Pacher

Error. Please check if input file exists
```
Using absolute path works:

```
$ haplogrep classify --format vcf --extend-report --in /path/to/vcf/test.vcf --out test.txt

https://haplogrep.i-med.ac.at
Sebastian Schoenherr, Hansi Weissensteiner, Lukas Forer and Domininc Pacher
Institute of Genetic Epidemiology, Medical University of Innsbruck
Contact: sebastian.schoenherr@i-med.ac.at
Welcome to HaploGrep v2.2.3
Instiute of Genetic Epidemiology, Medical University of Innsbruck
Hansi Weissensteiner, Sebastian Schönherr, Lukas Forer, Dominic Pacher

phylotree17.xml
Parameters:
Input Format: vcf
Phylotree Version: 17
Reference: rCRS
Extended Report: true
Fix Nomenclature: false
Used Metric: kulczynski
Chip array data: false
Lineage: false

Start Classification...
HaploGrep file written to /path/to/haplogrep/test.txt (Time: 0 sec)
```
It looks like if haplogrep is considering its installation path as the working directory. In fact, as you see in the second example (absolute input path, relative output path), the output file is written there. Of course it all works if you launch haplogrep from its folder, or if the vcf file is next to the haplogrep binary.

In case it has to be with the OS, I'm working on a Red Hat environment:
```
Operating System: Red Hat Enterprise Linux Server 7.5 (Maipo)
CPE OS Name: cpe:/o:redhat:enterprise_linux:7.5:GA:server
Kernel: Linux 3.10.0-862.11.6.el7.x86_64
```

... plus the "Instiute" typo is still there. It is correct in the contact information, but not after the welcome message.

**Comment by seppinho** [comment]
Thanks for the description! Issue has now been fixed with help from @lukfor. Feel free to close the issue.
Best and thanks again.
Sebastian

**Comment by jamigo** [comment]
Confirmed. Issue solved in v2.2.5. Thank you.
